gtkmm.json

```diff
@@ -64,9 +73,9 @@
             "name": "glibmm",
             "sources": [
                 {
-                    "sha256": "9e1db7d43d2e2d4dfa2771354e21a69a6beec7c446b711619cf8c779e13a581e",
+                    "sha256": "38684cff317273615c67b8fa9806f16299d51e5506d9b909bae15b589fa99cb6",
                     "type": "archive",
-                    "url": "https://ftp.gnome.org/pub/GNOME/sources/glibmm/2.66/glibmm-2.66.0.tar.xz"
+                    "url": "https://download.gnome.org/sources/glibmm/2.82/glibmm-2.82.0.tar.xz"
```

Comment:
not sure why you try to blindly upgrade everything to the latest, but glibmm 2.66 is the last version that is compatible with gtkmm 3.24. 2.68 broke API and is only for gtkmm4.

Latest is not always best.

Reply:
hey Hubert, thanks for your guidance, I'm not savvy in gtkmm bindings release policy and I wrongly thought I could update to latest. Sooner or later I would have found out the build won't succeed, but thank you for letting me know in advance.